### ISSUE TITLE
Items are not displayed on the home page

### ISSUE THREAD

**Issue by PierreClementCazon** [diomed/casper#7]
Hi,

I can't display the items on the home page of my site, here is the current configuration: 

![1](https://user-images.githubusercontent.com/36991574/49685391-e6691780-fae2-11e8-9c51-6482bfa820f6.PNG)
![2](https://user-images.githubusercontent.com/36991574/49685392-e6691780-fae2-11e8-87c2-f5340cb6048a.PNG)
![3](https://user-images.githubusercontent.com/36991574/49685393-e701ae00-fae2-11e8-9159-c6e0b017a21b.PNG)
![4](https://user-images.githubusercontent.com/36991574/49685394-e701ae00-fae2-11e8-8c51-8ae74b45756f.PNG)

A separate question, how to create a child theme to overload some files in the theme?

Thank you in advance for your help!

**Comment by diomed** [comment]
your items should be **subpages** [children] of blog.
that's how you populate it.

---
this article is useful for creating child themes:

https://getgrav.org/blog/theme-development-with-inheritance

**Comment by PierreClementCazon** [comment]
Same result with a subpage and the use of self.children in blog config :

![1_2](https://user-images.githubusercontent.com/36991574/49686107-0226ea80-faf0-11e8-8c53-ef160f69fbf7.PNG)
![1_3](https://user-images.githubusercontent.com/36991574/49686108-02bf8100-faf0-11e8-903f-e453815a0083.PNG)
![1_4](https://user-images.githubusercontent.com/36991574/49686109-02bf8100-faf0-11e8-942a-7d4f35fcf773.PNG)

**Comment by PierreClementCazon** [comment]
Yes I tried these two ways but I edited this parameter well with the new page structure.

![image](https://user-images.githubusercontent.com/36991574/49686699-ac0a7500-faf8-11e8-9bf0-f90cbba465b1.png)

**Comment by diomed** [comment]
🤷‍♂️ could you contact me on discord?

ameo #9752

**Comment by PierreClementCazon** [comment]
Rather by mail if you can give me yours?
